internal/app/helm_helpers.go

```diff
@@ -4,21 +4,28 @@ import (
 	"encoding/json"
 	"errors"
 	"fmt"
+	"gopkg.in/yaml.v2"
 	"net/url"
 	"path/filepath"
-	"regexp"
 	"strings"
 
 	"github.com/hashicorp/go-version"
 
 	"github.com/Praqma/helmsman/internal/gcs"
+
+	"github.com/Masterminds/semver/v3"
```

Comment:
in other places we're using `github.com/hashicorp/go-version` can you use that here as well? or change the other places to use this one....

Reply:
Looks like it has the exact same methods, I'll switch to go-version